manifests/02-crd.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: tuneds.tuned.openshift.io
+spec:
+  group: tuned.openshift.io
+  names:
+    kind: Tuned
+    listKind: TunedList
+    plural: tuneds
```

Comment:
idk enough about crd - this just seems odd

Reply:
Welcome to the club, this is exactly what the operator-sdk gave me, and I've adopted it for the moment.